include/openspace/util/time.h

```diff
@@ -141,6 +141,20 @@ class Time {
      */
     double advanceTime(double tickTime);
 
+    /**
+     * Sets a relative time from profile.
+     * \param setTime a string containing time adjustment as described in documentation
+     * for luascriptfunctions::time_advancedTime
+     */
+    void setFromProfile_timeRelative(const std::string& setTime);
```

Comment:
-> setRelativeTimeFromProfile  ?

Likewise for the function below

Reply:
Changed